pyproject.toml

```
@@ -7,7 +7,7 @@ name = "Django"
 dynamic = ["version"]
 requires-python = ">= 3.10"
 dependencies = [
-    "asgiref>=3.7.0",
+    "asgiref>=3.8.1",
```

This line should be backported to `stable/5.1.x` with `<4`
```suggestion
    "asgiref>=3.8.1,<4",
```